Documentation/install/vmware/vmware-terraform.md

```diff
@@ -57,7 +58,7 @@ $ ssh-add -L
 Reference the absolute path of the **_public_** component of the SSH key in `tectonic_vmware_ssh_authorized_key`.
 
 Without this, terraform is not able to SSH copy the assets and start bootkube.
-Also make sure that the SSH known_hosts file doesn't have old records of the API DNS name (fingerprints will not match).
+Also make sure that the SSH `known_hosts` file doesn't have old records of the API DNS name (fingerprints will not match).
```

Comment:
Replace with: "Ensure the SSH `known_hosts` file does not contain old records for the API DNS name to avoid a key fingerprint mismatch."